plugins/omar-wms-plugin/grails-app/services/omar/wms/WebMappingService.groovy

```diff
@@ -268,12 +285,18 @@ class WebMappingService implements InitializingBean
 
   def getMap(GetMapRequest wmsParams)
   {
-    def otherParams = [startDate: new Date()]
+    def otherParams = [startTime: new Date(), internalTime: new Date(), procTime: new Date()]
 
     log.trace "getMap: Entered ................"
     otherParams.startTime = System.currentTimeMillis()
     otherParams.internalTime = otherParams.startTime
 
+    // log timestamp (can be added to otherParams if needed)
+    TimeZone.getTimeZone('UTC')
```

Comment:
What is the purpose of this getTimeZone call? The function returns a static TimeZone object that is not being assigned in the return. https://docs.oracle.com/javase/7/docs/api/java/util/TimeZone.html

Reply:
Got rid of it.